R/reexport.R

```diff
@@ -0,0 +1,8 @@
+#' @importFrom piamutils substitutePlaceholder
+#' @export
```

Comment:
not sure we need the substitutePlaceholder function, because its name was substituePlaceholder with a typo anyway and I think nobody used it outside remind2.

Reply:
correct, i will remove it